### ISSUE TITLE
Pointbank Tagline

### ISSUE THREAD

**Issue by apforsyth** [wicksipedia/PointBank#17]
Hi all,

Suggest changing the tagline for the app from:

 - Bringing high interest rates to boring tasks
To:

- Making boring tasks easier!
My reason is that "high interest rates", while being a cute word play, may have a negative connotation, and when coupled with the word "boring", with a kind of double negative connotation, doesn't really sell the benefit as strongly or positively as possible.

Other more positive variants could be:

Make boring tasks easy and compete for rewards! (a positive call to action)
Making the mundane more meaningful (a little alliteration to help it roll off the tongue ;))
Any other ideas?

**Comment by calumjs** [comment]
Agreed it could definitely be improved, I just wanted to get my word play in there for the initial version ;)

**Comment by apforsyth** [comment]
Ha!  great video presentation overall, thanks for introducing me to Github
